webapp_frontend/src/background/constants.tsx

```diff
@@ -0,0 +1,18 @@
+
+interface constants {
+    url: { API_URL: string }
+}
+
+
+const prod: constants = {
+    url: {
+        API_URL: window.location.hostname,
+    }
+}
+
+const dev: constants = {
+    url: {
+        API_URL: 'http://filefighter.ddns.net:7000/',
+    }
+};
```

Comment:
why this. why not localhost:80 when using nxginx?

Reply:
This is only applied when using npm start. So for convenience of FE Devs it uses the server as the BE

Reply:
Ok, maybe if we add `filefighter start` to `npm start` it could be localhost.